test/state_test.js

```diff
@@ -79,7 +79,7 @@ describe('State', () => {
 
     it('one flat condition', () => {
       let expected = {
-        text: 'SELECT * FROM junk WHERE junk.a = $1;',
+        text: 'SELECT * FROM junk WHERE junk."a" = $1;',
```

Comment:
Just out of curiosity: why the quotation marks?

Reply:
In this specific case it's overkill, but they're used for case-sensitivity.  It was easier to just quote everything even if unneeded than to determine if there were caps in the column name and only quote then.  Really though, in both cases, we should be using parameterized condition names.